.github/workflows/app_check.yml

```diff
@@ -21,7 +21,6 @@ jobs:
     strategy:
       matrix:
         target: [ios, tvos, macos]
-    # TODO(v9): Remove `env` after publishing `HeartbeatLoggingTestUtils`.
     env:
       POD_LIB_LINT_ONLY: 1
```

Comment:
Do you mind keeping a comment above the `POD_LIB_LINT_ONLY` here and below mentioning that there's a dependency on the unpublished `HeatbeatLoggingTestUtils`? That way when we come back to it in a year or so we aren't asking ourselves why is this lib lint only :)